src/cmds/testing/spi/Mybuild

```diff
@@ -0,0 +1,30 @@
+package embox.cmd.testing
+
+@AutoCmd
+@Cmd(name = "spitest",
```

Comment:
by the way, do you see the 'spi' command (src/cmds/hardware/spi)
https://github.com/embox/embox/tree/master/src/cmds/hardware/spi

Reply:
I'll move the functionality over to that one.